src/components/Reports/GDPRPermissions/SubscriberGdprPermissions.js

```diff
@@ -52,7 +54,7 @@ const SubscriberGdprPermissions = ({
     <div>
       <div className="dp-table-responsive">
         {state.loading ? (
-          <Loading />
+          <Loading page />
```

Comment:
Is it fine?

Reply:
In my opinion the loading is showed better in that way.